meson.build

```diff
@@ -124,6 +124,14 @@ else
 endif
 have_getopt_long = cc.has_header_symbol('getopt.h', 'getopt_long',
                                         prefix: '#define _GNU_SOURCE')
+if not cc.has_header_symbol('limits.h', 'PATH_MAX', prefix: system_ext_define)
+    if cc.get_id() == 'msvc'
```

Comment:
Since it is an OS definition, would `host_machine.system() == 'windows'` be more appropriate?

Reply:
fixed. I guess you can tell I'm not doing a lot of cross-platform development...